pyvelox/pyvelox.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include "pyvelox.h" // @manual
+#include "fuzzer.h" // @manual
```

Comment:
what's the @ manual about?

Reply:
In fbcode , pybind + buck requires this otherwise builds fail .